src/apps/api/views/competitions.py

```diff
@@ -96,6 +102,41 @@ def get_files(self, request, pk):
         return Response(files)
 
 
+@api_view(['GET'])
+@permission_classes((AllowAny,))
+def front_page_competitions(request):
+    popular_comps = get_popular_competitions()
+    featured_comps = get_featured_competitions(excluded_competitions=popular_comps)
+    popular_comps_serializer = CompetitionSerializerSimple(popular_comps, many=True)
+    featured_comps_serializer = CompetitionSerializerSimple(featured_comps, many=True)
+    return Response(data={
+        "popular_comps": popular_comps_serializer.data,
+        "featured_comps": featured_comps_serializer.data
+    }, status=status.HTTP_200_OK)
+
+
+@api_view(['GET'])
+@permission_classes((AllowAny,))
+def by_the_numbers(request):
+    total_competitions = Competition.objects.all().count()
+    public_competitions = Competition.objects.filter(published=True).count()
+    private_competitions = Competition.objects.filter(published=False).count()
+    users = User.objects.all().count()
+    competition_participants = CompetitionParticipant.objects.all().count()
+    submissions = Submission.objects.all().count()
+    organizers = Competition.objects.values_list('created_by').distinct().count()
+
+    return Response(data={
```

Comment:
don't need the `data=`
Also, thinking we might want to restructure this response data too. so we don't have to do as much transforming in JS.
```python
return Response([
    {'label': 'Total Competitions', 'count': total_competitions},
    {'label': 'Public Competitions', 'count': public_competitions},
    # etc
])
```
See note below and how to handle the response.

Reply:
Really clean, I like this solution